how-to-build-gcc-from-source.rst

```diff
@@ -0,0 +1,133 @@
+How to Build GCC from Source
+============================
+
+Step 1: Install Dependancies
+----------------------------
+
```

Comment:
Needs something like "The precise command to install dependencies will vary between operating systems.  On Fedora..." or somesuch